shared/cfg/guest-os/Linux/RHEL/8.cfg

```diff
@@ -0,0 +1,58 @@
+- 8:
+    variants:
+        - aarch64:
+            vm_arch_name = aarch64
+        - ppc64:
+            vm_arch_name = ppc64
+        - ppc64le:
+            vm_arch_name = ppc64le
+        - s390x:
+            vm_arch_name = s390x
+        - x86_64:
+            vm_arch_name = x86_64
+    os_variant = rhel8
+    unattended_install.url:
+        #Running from http server, RHEL requires more memory
+        mem = 4096
+    nic_hotplug:
+        modprobe_module =
+    block_hotplug:
+        modprobe_module =
+    unattended_install, svirt_install:
+        kernel = images/${os_variant}-${vm_arch_name}/vmlinuz
+        initrd = images/${os_variant}-${vm_arch_name}/initrd.img
+        unattended_install, check_block_size.4096_512, check_block_size.512_512, svirt_install:
+            cdrom_unattended = images/${os_variant}-${vm_arch_name}/ks.iso
+            syslog_server_proto = udp
+        # ARCH dependent things
+        aarch64:
+            grub_file = /boot/grub/grub.conf
+            install_timeout = 7200
+        ppc64:
+            no guest_s3, guest_s4
+            mem_chk_cmd = numactl --hardware | awk -F: '/size/ {print $2}'
+            netdev_peer_re = "(.*?): .*?\\\s(.*?):"
+            unattended_install..floppy_ks:
+                floppies = "fl"
+                floppy_name = images/${os_variant}-${vm_arch_name}/ks.vfd
+        ppc64le:
+            no guest_s3, guest_s4
```

Comment:
Would be good to double-check whether suspend/hibernate is supported in RHEL.8 (it was not in RHEL.7, which is the reason for those...)

Reply:
I can suspend/hibernate VM according to https://www.linux-kvm.org/page/PowerManagement. I tested this, but I was not able to wake up VM afterwards (maybe to send some key is not enough - do not know). There is also possibility to suspend and resume VM using `virsh`. I tested this and for RHEL.8 x86_64 it works. I did not find any official documentation regarding (un)support of suspend/hibernate. So I am not sure about that.

Reply:
The virsh suspend should be a different feature. Anyway see the email from David, it is not supported so please keep it here until someone asks it to enable this.